resources/product.py

```diff
@@ -16,6 +19,7 @@
 
 product_args = reqparse.RequestParser()
 product_args.add_argument("_id", type=str, help="Problem with Product ID value")
+product_args.add_argument("user_id", type=str, help="Problem with User ID value")
```

Comment:
Either create a new reqparse object so we can use user_id when creating a group, or just use request.